Sources/Purchasing/StoreKitAbstractions/StoreEnvironment.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  StoreEnvironment.swift
+//
+//  Created by MarkVillacampa on 26/10/23.
+//
+
+import Foundation
+import StoreKit
+
+/// A wrapper for `StoreKit.AppStore.Environment`.
+enum StoreEnvironment: String {
```

Comment:
I see you're using `: String` for the `Codable` implementation. Are you using the `Codable` implementation for this type though?
I can't really find where it's used. But if so, maybe add a comment mentioning what the `rawValue` is used for? Basically We want to make sure we don't inadvertently change it. Alternatively if we are using it somewhere like a request, it would be useful to add it to the snapshot tests to capture that.

Reply:
It's used when serializing `StoreKit2Receipt`. Added snapshot tests for the request where it's used